pkg/rpc/client.go

```diff
@@ -56,26 +56,25 @@ type ClientConfig struct {
 	L2EngineEndpoint      string
 	JwtSecret             string
 	RetryInterval         time.Duration
-	Timeout               *time.Duration
-	BackOffMaxRetrys      *big.Int
+	Timeout               time.Duration
+	BackOffMaxRetry       uint64
```

Comment:
Shall we use `Retries`? ref: https://pkg.go.dev/github.com/cenkalti/backoff/v4@v4.1.3#WithMaxRetries